### ISSUE TITLE
Scoped packages by default

### ISSUE THREAD

**Issue by dom96** [nim-lang/nimble#452]
I believe this was already requested by some people. The more I think about it the more I believe I should enforce it.

Here is npm's description of what scoped packages are: https://docs.npmjs.com/misc/scope

Any concerns?

**Comment by genotrance** [comment]
I think this ties back to the namespace debate and the outcome was to just leave it as is. Closing.

**Comment by dom96** [comment]
Nah, this is different and worthy of discussion too. Especially in the context of how Nimble treats package structures. Also I'm not sure we've decided yet to leave things as-is, people don't seem to be happy with the status quo after all.
